<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DownloadFromDiveComputer</class>
 <widget class="QDialog" name="DownloadFromDiveComputer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>535</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Download from dive computer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:subsurface-icon</normalon>
   </iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="choicesLayout">
         <property name="horizontalSpacing">
          <number>0</number>
         </property>
         <property name="verticalSpacing">
          <number>5</number>
         </property>
         <item row="7" column="0">
          <widget class="QCheckBox" name="forceDownload">
           <property name="text">
            <string>Force download of all dives</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Dive computer</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QComboBox" name="vendor"/>
         </item>
         <item row="8" column="0">
          <widget class="QCheckBox" name="preferDownloaded">
           <property name="text">
            <string>Always prefer downloaded dives</string>
           </property>
          </widget>
         </item>
         <item row="10" column="0">
          <widget class="QCheckBox" name="logToFile">
           <property name="text">
            <string>Save libdivecomputer logfile</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QComboBox" name="device">
           <property name="editable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="12" column="0">
          <widget class="QCheckBox" name="bluetoothMode">
           <property name="text">
            <string>Choose Bluetooth download mode</string>
           </property>
          </widget>
         </item>
         <item row="12" column="1">
          <widget class="QToolButton" name="chooseBluetoothDevice">
           <property name="toolTip">
            <string>Select a remote Bluetooth device.</string>
           </property>
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QToolButton" name="search">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="11" column="1">
          <widget class="QToolButton" name="chooseDumpFile">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Device or mount point</string>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QToolButton" name="chooseLogFile">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Vendor</string>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QCheckBox" name="createNewTrip">
           <property name="text">
            <string>Download into new trip</string>
           </property>
          </widget>
         </item>
         <item row="11" column="0">
          <widget class="QCheckBox" name="dumpToFile">
           <property name="text">
            <string>Save libdivecomputer dumpfile</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QComboBox" name="product"/>
         </item>
         <item row="6" column="0">
          <layout class="QGridLayout" name="DCGrid">
           <item row="0" column="0">
            <widget class="QFrame" name="DCFrame1">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="DC1Grid">
              <property name="horizontalSpacing">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QPushButton" name="DC1">
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="DeleteDC1">
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
                <property name="icon">
                 <iconset>
                   <normaloff>:list-remove-icon</normaloff>:list-remove-icon</iconset>
                 </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>

           <item row="0" column="1">
            <widget class="QFrame" name="DCFrame2">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="DC2Grid">
              <property name="horizontalSpacing">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QPushButton" name="DC2">
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="DeleteDC2">
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
                <property name="icon">
                 <iconset>
                   <normaloff>:list-remove-icon</normaloff>:list-remove-icon</iconset>
                 </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>

           <item row="1" column="0">
            <widget class="QFrame" name="DCFrame3">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="DC3Grid">
              <property name="horizontalSpacing">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QPushButton" name="DC3">
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="DeleteDC3">
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
                <property name="icon">
                 <iconset>
                   <normaloff>:list-remove-icon</normaloff>:list-remove-icon</iconset>
                 </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>

           <item row="1" column="1">
            <widget class="QFrame" name="DCFrame4">
             <property name="frameShape">
              <enum>QFrame::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <layout class="QGridLayout" name="DC4Grid">
              <property name="horizontalSpacing">
               <number>0</number>
              </property>
              <item row="0" column="0">
               <widget class="QPushButton" name="DC4">
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="DeleteDC4">
                <property name="autoDefault">
                 <bool>false</bool>
                </property>
                <property name="flat">
                 <bool>true</bool>
                </property>
                <property name="icon">
                 <iconset>
                   <normaloff>:list-remove-icon</normaloff>:list-remove-icon</iconset>
                 </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>

          </layout>
         </item>
         <item row="13" column="0">
          <widget class="QCheckBox" name="syncDiveComputerTime">
           <property name="text">
            <string>Sync dive computer time</string>
           </property>
           <property name="toolTip">
            <string>Adjust the time on the dive computer to match the time on the PC (if supported by the dive computer model).</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="downloadCancelRetryLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="downloadCancelRetryButton">
           <property name="text">
            <string>Download</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="progressText">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="aboveOKCancelSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="labelSelectAllNoneLayout">
         <property name="spacing">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="downloadedDivesLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>1</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Downloaded dives</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="selectAllButton">
           <property name="text">
            <string>Select all</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="unselectAllButton">
           <property name="text">
            <string>Unselect all</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="downloadedView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonBoxLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="ok">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
